### DISCUSSION TITLE
Running Flood with existing qBittorrent containers in Docker

### DISCUSSION THREAD

**Discussion by epicylon** [jesec/flood#282] [Q&A] [answered]
Hello,

I have a few Docker containers that have been running qBittorrent for a while. Is it possible to "implement" Flood with these containers? 

If it as at all possible, I would gladly accept some input on how it can be done.

**Comment by jesec** [comment, marked as answer]
Just run Flood, share folders with the same contexts, and allow Flood to connect to qBittorrent's API.
